### ISSUE TITLE
PHPCBF can make invalid fixes to inline php control structures with try catch

### ISSUE THREAD

**Issue by ST-o-RM** [squizlabs/PHP_CodeSniffer#1477]
Hello,

InlineControlStructure CBF set brace after "try" forgetting about "catch":

```
-		if (!empty($campaignIds))
+		if (!empty($campaignIds)) {
 			try {
 				$accountChanges = $customerSyncService->get($selector);
-			} catch (Exception $e) {
+			}
+		} catch (Exception $e) {
 					return TRUE;
 			}
```

PHP_CodeSniffer version 3.0.0RC2 (beta)

**Comment by jrfnl** [comment]
What version of PHPCS are you using ? Please check with `phpcs --version`.
I cannot currently reproduce the issue. It looks like this was already fixed in January - see: #1253.

**Comment by gsherwood** [comment]
Finally got around to testing this. Using 3.0.0RC2, I get this diff using the PSR2 standard:
```diff
 <?php
-if (!empty($campaignIds))
+if (!empty($campaignIds)) {
     try {
         $accountChanges = $customerSyncService->get($selector);
-    } catch (Exception $e) {
-        return TRUE;
     }
+} catch (Exception $e) {
+    return true;
+}
```
Using the latest 3.0.1 release, I get this diff:
```diff
 <?php
-if (!empty($campaignIds))
+if (!empty($campaignIds)) {
     try {
         $accountChanges = $customerSyncService->get($selector);
     } catch (Exception $e) {
-        return TRUE;
+        return true;
     }
+}
```
So I can confirm it was a bug in RC2, but was fixed in 3.0.1 and has been released. Can you test the latest version to see if it is working for you?
